cluster/src/dunit/scala/io/snappydata/externalstore/ColumnTableDUnitTest.scala

```diff
@@ -160,8 +175,7 @@ class ColumnTableDUnitTest(s: String) extends ClusterManagerTestBase(s) {
     val p = Map.empty[String, String]
     snc.createTable(tableName, "column", dataDF.schema, p)
 
-    val tName = s"${Constant.INTERNAL_SCHEMA_NAME}.${
-      tableName.toUpperCase}${Constant.SHADOW_TABLE_SUFFIX}"
+    val tName = ColumnFormatRelation.cachedBatchTableName(tableName.toUpperCase())
```

Comment:
@nthanvi Missed this in initial review. Both original and updated code should never be using toUpperCase which breaks case-sensitivity SQLConf setting. Can you check precheckin removing toUpperCase? I believe all case-sensitivity issues should be now fixed in current code.